vendor/bat-native-ads/src/bat/ads/internal/client.h

```diff
@@ -86,6 +86,8 @@ class Client {
       const std::string& campaign_id);
   const std::map<std::string, std::deque<uint64_t>>
       GetCampaignHistory() const;
+  std::string GetVersionCode() const;
```

Comment:
please add `const` before `std::string`

Reply:
It's returned by value not by ref. What the reason to add `const`?

Reply:
Ah yes, my error